docs/products/kafka/howto/approvals.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Approvals
+limited: true
+---
+
+The approvals page allows you to manage ownership requests for Aiven for Apache Kafka resources owned by your group.
+
+These requests can include claims for existing topics or requests to create new topics.
+You can review and process these requests to either approve or decline them.
+
+
+## Requests flow
+
+Managing requests involves several key steps:
+
+- **Request initiation**: Users submit a request from the Apache Kafka topic catalog to
+  claim ownership of an existing topic or create a topic.
+- **Request visibility**: All members of the topic's owning group can see the request on
+  the approvals page.
+- **Review and approval**: A group member reviews the request, and either approves or
+  declines it.
+- **Completion**: If approved, the requested topic ownership change or topic creation
+  is completed. If declined, the request is not implemented.
+
+## Key elements
+
+The approvals page displays the following key elements:
+
+- **Topic**: Name of the topic
+- **Type**: Claim or create request
+- **Status**: Pending, approved, or declined
```

Comment:
Deleted might be good to add in here for when you delete your own request.

We also have a Provisioning state and a Failed state as well.

Provisioning is when the Topic is being created on the cluster.
Failed is when the request was unable to complete the governance action. It could be because the kafka cluster is down or because of a connectivity issue etc.

Reply:
I just realized this is the approvals, I don;t know if any other statuses actually show up I need to check.
But if they dont we should remove them from the drop down in the UI ^^

Reply:
ok double checked, technically all status' can be seen by the approver.

Reply:
Thanks, added the remaining statuses.